HedgeModManager/HedgeApp.xaml.cs

```diff
@@ -160,7 +160,10 @@ public static void Main(string[] args)
 
             // Include commit hash if defined
             if (!string.IsNullOrEmpty(RepoCommit))
+            {
+                ProgramName += $" Development";

```

Comment:
Should remove the dollar sign there since it's not an interpolated string.

Reply:
Fixed.